changelog.d/config

```diff
@@ -1,3 +1,19 @@
 organization: haskell
 repository:   cabal
 required-fields: packages prs
+packages:
+  Cabal
```

Comment:
Changelogs would only be needed for `Cabal-syntax`, `Cabal-hooks`, `Cabal`, `cabal-install-solver`, and `cabal-install`; the others are internal.